mdx/guides/understanding-rfqt.mdx

```diff
@@ -0,0 +1,62 @@
+
+Trusted takers are identified by an API key, specified in the HTTP header `0x-api-key`. In addition, to qualify for RFQ-T liquidity, the request must also specify a non-null `takerAddress` query parameter, which will be included in the order that the maker signs, preventing anyone else from filling that order.
+
+The 0x API can be configured with a list of API keys which are permitted to access RFQ-T liquidity. For the instance at `api.0x.org`, the 0x team is maintaining a list of trusted API integrators, which at this time includes such projects as DeFi Saver and Zerion, and of course 0x's own Matcha.
+
+## Dedicated Makers
+
+In addition to the 0x API configuration identifying trusted takers, it also contains a list of specific market makers that participate in the RFQ-T model. Each maker is identified by an HTTP API endpoint URL, and each endpoint has an associated list of asset pairs for which that endpoint will provide quotes.
+
+For the instance at `api.0x.org`, the 0x team is maintaining a list of trusted market makers, which at this time includes 0x's own Periscope Trading, as well as a handful of other trusted partners.
+
+Whenever a trusted taker reqeuests a quote for an asset pair known to be served by at least one RFQ-T maker, the API's response will take RFQ-T liquidity into consideration.
+
+## Quote types: indicative vs firm
+
+In order to improve capital efficiency, the RFQ-T model introduces a new request/response flow: first request pricing, then request a fillable quote.
+
+When market makers provide a quote that can be settled, in the form of a 0x order that can be filled by submitting it to the blockchain, the taker of that order has an expectation that the swap will actually settle (that it won't revert due to lack of available maker assets). In order to uphold this expectation, makers should refrain from using those same assets as the basis for any other quotes, for the duration of the order.
+
+However, if every price inquiry required the maker to reserve funds, they would quickly end up with all of their funds reserved, and they would be unable to provide any other quotes until those assets were available again.
+
+To aleviate this potential capital inefficiency, the RFQ-T model introduces the distinction between "indicative pricing" and "firm quotes."
+
+The 0x API offers two different endpoints for the use of RFQ-T. One is an "indicative pricing" endpoint, hosted at `/swap/vX/price`. This endpoint responds with pricing information, but that response does not contain a full 0x order, so it does not constitute a legitimate transaction that can be submitted to the blockchain. Therefore, when a market maker services an indicative request they do not need to reserve any funds for it, leaving their funds available for other quotes to other clients.
```

Comment:
I see what you are doing with `vX`, however, do you think it's worth specifying somewhere that `vX` is the latest version. Just to ensure 100% no ambiguity?